migrations/1597654308406-remove-importHashes-from-users-with-accounts.js

```diff
@@ -30,20 +29,27 @@ module.exports = {
 				.sort({
 					updatedAt: 1,
 					createdAt: 1,
-				}).skip(looped)
+				})
+				.skip(looped)
 				.limit(limit)
 				.lean()
 				.exec();
 			// eslint-disable-next-line no-loop-func
-			await Promise.all(users.map((user) => accountModel.find({ userId: user._id }).countDocuments()
-				.then((accounts) => {
-					if (accounts > 0) {
-						user.importHash = null;
-						removedImportHashes += 1;
-						return userModel.findByIdAndUpdate(user._id, user);
-					}
-					return Promise.resolve();
-				})));
+			await Promise.all(
+				users.map((user) =>
```

Comment:
Codacy found an issue: [Function declared in a loop contains unsafe references to variable(s) 'removedImportHashes'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)